### ISSUE TITLE
Output of flow_legacy throws exception for bubble point and dew point pressure

### ISSUE THREAD

**Issue by blattms** [OPM/opm-simulators#1144]
For model 2 I get the following exception:
```
Time step    0 at day 0/4870, date = 01-Jan-2000
  Substep 0, stepsize 0.66 days.
    Substep summary: newton its =  1, linearizations =  2 ( 3.192 sec), linear its =   1 ( 0.284 sec)
Error: Program threw an exception: The cell data with name: PBUB does not exist
```
I have not idea how to fix this but IMHO it should be fixed for the release.

**Comment by blattms** [comment]
Happens for Norne, too.

**Comment by blattms** [comment]
Deleted last comment. As I used a patch version of flow_legacy. With the master version there is no problem for Norne. Just for Model 2.

**Comment by andlaus** [comment]
flow_mpi is also affected by this, but flow_ebos works fine.

**Comment by blattms** [comment]
Solved a long time ago by #1147
